src/coreComponents/mesh/generators/WellGeneratorBase.cpp

```diff
@@ -528,54 +529,62 @@ void WellGeneratorBase::debugWellGeometry() const
+  std::vector< string > row;
+  std::ostringstream oss;
+  Table table = Table( {
+      Table::ColumnParam{{"Element no."}, Table::Alignment::right},
+      Table::ColumnParam{{"CoordX"}, Table::Alignment::middle},
+      Table::ColumnParam{{"CoordY"}, Table::Alignment::middle},
+      Table::ColumnParam{{"CoordZ"}, Table::Alignment::middle},
+      Table::ColumnParam{{"Prev\nElement"}, Table::Alignment::right},
+      Table::ColumnParam{{"Next\nElement"}, Table::Alignment::right},
+    }
+                       );
+
+  string titleName = "InternalWellGenerator " + getName();
+  table.setTitle( titleName );
 
   for( globalIndex iwelem = 0; iwelem < m_numElems; ++iwelem )
   {
-    std::cout << "Well element #" << iwelem << std::endl;
-    std::cout << "Coordinates of the element center: " << m_elemCenterCoords[iwelem] << std::endl;
-    if( m_nextElemId[iwelem] < 0 )
-    {
-      std::cout << "No next well element" << std::endl;
-    }
-    else
-    {
-      std::cout << "Next well element # = " << m_nextElemId[iwelem] << std::endl;
-    }
-    if( m_prevElemId[iwelem][0] < 0 )
+    std::optional< globalIndex > nextElement;
+    std::optional< globalIndex > prevElement;
+
+    if( m_nextElemId[iwelem] >= 0 )
     {
-      std::cout << "No previous well element" << std::endl;
+      nextElement =  m_nextElemId[iwelem];
     }
-    else
+
+    if( m_prevElemId[iwelem][0] >= 0 )
     {
-      std::cout << "Previous well element #" << m_prevElemId[iwelem][0] << std::endl;
+      prevElement =  m_prevElemId[iwelem][0];
     }
+
     for( globalIndex inode = 0; inode < m_numNodesPerElem; ++inode )
     {
       if( inode == 0 )
       {
-        std::cout << "First well node: #" << m_elemToNodesMap[iwelem][inode] << std::endl;
+        //std::cout << "First well node: #" << m_elemToNodesMap[iwelem][inode] << std::endl;
       }
       else
       {
-        std::cout << "Second well node: #" << m_elemToNodesMap[iwelem][inode] << std::endl;
+        //std::cout << "Second well node: #" << m_elemToNodesMap[iwelem][inode] << std::endl;
       }
     }
+    table.addRow( iwelem, m_elemCenterCoords[iwelem][0], m_elemCenterCoords[iwelem][1], m_elemCenterCoords[iwelem][2], prevElement, nextElement );
+
   }
+  table.draw();
 
-  std::cout << std::endl << "Number of perforations = " << m_numPerforations << std::endl;
+  Table tablePerforation = Table( {"Perforation no.", "Coordinates\nlong string", "connected to" } );
+  string titlePerfo = "Peforation table";
+  tablePerforation.setTitle( titlePerfo );
 
   for( globalIndex iperf = 0; iperf < m_numPerforations; ++iperf )
   {
-    std::cout << "Perforation #" << iperf << std::endl;
-    std::cout << "Coordinates of the perforation: " << m_perfCoords[iperf] << std::endl;
-    std::cout << "Is connected to well element #" << m_perfElemId[iperf] << std::endl;
+    tablePerforation.addRow( iperf, m_perfCoords[iperf], m_perfElemId[iperf] );
   }
-  std::cout << std::endl;
+
+  tablePerforation.draw();
```

Comment:
Suggestion : In a later PR, in order to output perforations transmissibilities, this table should be output in `CoupledReservoirAndWellsBase::computeWellTransmissibility()`.